### ISSUE TITLE
Getting error Object literal may only specify known properties, and 'boxplot' does not exist in type '_DeepPartialObject<ElementOptionsByType<"boxplot">>' during GitHub build

### ISSUE THREAD

**Issue by maximgorbatyuk** [sgratzl/chartjs-chart-boxplot#136]
I use the library for my open source web project written with Angular v17, and did not change anything regarding charts today. But, during the build I got the error `Object literal may only specify known properties, and 'boxplot' does not exist in type '_DeepPartialObject<ElementOptionsByType<"boxplot">>'.` by some reason. I don't know to what is it related to, but it started to happen. Could you please have a look into error.

**To Reproduce**

1. Open https://github.com/Techinterview-space/frontend/actions/runs/9436083106/job/25990331074
2. Go to build error

**Expected behavior**

No errors like in previous builds (https://github.com/Techinterview-space/frontend/actions/runs/9429276168/job/25975503541)

**Action result**:

The error `Object literal may only specify known properties, and 'boxplot' does not exist in type '_DeepPartialObject<ElementOptionsByType<"boxplot">>'.` appears. File https://github.com/Techinterview-space/frontend/blob/ce40307f2ac08edbddfef6e5e04d391a321a4690/src/app/modules/salaries/components/grades-min-max-salaries-chart/grades-min-max-chart-object.ts#L70

**Screenshots**

![CleanShot 2024-06-09 at 16 15 21](https://github.com/sgratzl/chartjs-chart-boxplot/assets/13348685/9726fb84-06cf-442f-8661-db2ef7ad6a49)

**Context**

- Version: 4.2.8
- Browser: none

**Comment by maximgorbatyuk** [comment]
Forgot to mention that the same build command works fine locally

**Comment by maximgorbatyuk** [comment]
Ok, I got the issue. Pipeline used command `nom update` which upgrades minor version of the library up to 4.4.0, but I was not doing it locally. After I did it, I also started to see the error:

![CleanShot 2024-06-10 at 09 02 11@2x](https://github.com/sgratzl/chartjs-chart-boxplot/assets/13348685/bb53f022-9795-4096-962b-70edcd972232)

Just removed the unknown property, and build was successful. Sorry for the issue. Closing it.
